src/woo_search/search_index/tests/test_analyzer.py

```diff
@@ -0,0 +1,32 @@
+from woo_search.search_index.tests.base import ElasticSearchTestCase
+
+from ...utils.tests.vcr import VCRMixin
+from ..client import get_client
+from ..index import Document, Publication
+
+
+class AnalyzerTest(VCRMixin, ElasticSearchTestCase):
+    def test_dutch_analyzer(self):
+        text = "Nederlandse tekst om de dutch analyzer te testen."
+        expected_tokens = [
+            "Nederlandse",
+            "tekst",
+            "om",
+            "de",
+            "dutch",
+            "analyzer",
+            "te",
+            "testen",
+        ]
```

Comment:
What would come up from the default analyzer if another analyzer would be used? I don't understand what this test is proving.